pkg/ccl/backupccl/testdata/backup-restore/online-restore-report-results

```diff
@@ -0,0 +1,27 @@
+# This test ensures that online restore provides relevant output
+# results.
+
+reset test-nodelocal
+----
+
+new-cluster name=s1 disable-tenant
+----
+
+exec-sql
+CREATE DATABASE d;
+USE d;
+CREATE TABLE foo (i INT PRIMARY KEY, s STRING);
+CREATE TABLE foofoo (i INT PRIMARY KEY, s STRING);
+INSERT INTO foofoo VALUES (10, 'x0');
+CREATE TABLE baz (i INT PRIMARY KEY, s STRING);
+INSERT INTO baz VALUES (1, 'x'),(2,'y'),(3,'z');
+----
+
+exec-sql
+BACKUP INTO 'nodelocal://1/cluster';
+----
+
+query-sql regex=(\d\d+\ssucceeded\s3\s[1-9]\d*\s[1-9]\d*\s\d\d+)
```

Comment:
I regret telling you to make this a data driven test. To make the results more readable, I think we ought to transform this into a go test. Take a look at similar test: `TestTenantStreamingShowTenant`

Reply:
I noticed that some of the Bazel Essential/Extended tests were failing so was looking into it, and saw that it was `pkg/ccl/backupccl/backuprand/backup_rand_test.go`. I'm updating those tests at the moment, but it looks like those are the types of tests we want?

Reply:
those tests are probably failing because they expect the old return results, so you'll need to teach these tests about your new return results.